### ISSUE TITLE
Bundle Quarto CLI with Positron

### ISSUE THREAD

**Issue by petetronic** [posit-dev/positron#3705]
The Quarto Extension does not come with a Quarto CLI, so to make it simpler for customers to use Quarto with Positron out of the box, we should also look to bundle the Quarto CLI.

We currently bundle the Pandoc CLI separately, this would need to be coordinated with the inclusion of Quarto instead.

**Comment by jmcphers** [comment]
We're bundling Quarto as of #4212, but to complete this story we also need a version of the Quarto extension that can find the bundle. https://github.com/quarto-dev/quarto/pull/503 contains the needed changes.

**Comment by coatless** [comment]
Looks like https://github.com/quarto-dev/quarto/pull/503 just was merged in by JJ. Any chance the built-in Quarto extension could get bumped? 😄

**Comment by jmcphers** [comment]
Yes, we'll bundle 1.114.0 of the extension when it becomes available (JJ will need to do the release first).

**Comment by jmcphers** [comment]
Should be ready to test now, note that there's still a problem with the Quarto extension not adding the Quarto path to the `PATH` if the Positron Quarto installation is your _only_ Quarto installation. This needs to be fixed in the Quarto extension.

https://github.com/quarto-dev/quarto/issues/517

**Comment by testlabauto** [comment]
Anyone have thoughts on what can be tested prior to https://github.com/quarto-dev/quarto/issues/517 being completed?  It seems to me like we might need to wait on that to really test that bundling works.  Without it, I am assuming that one would not truly be using the bundled extension.

**Comment by jonvanausdeln** [comment]
As I just posted in the quarto issue https://github.com/quarto-dev/quarto/issues/517#issuecomment-2316009206

I just tried with the latest build of Positron ([2024.08.0-77](https://github.com/posit-dev/positron-builds/releases/tag/2024.08.0-77) on Windows and it doesn't find Quarto.
* Uninstalled all quarto versions
* Uninstalled all Positron
* Installed [2024.08.0-77](https://github.com/posit-dev/positron-builds/releases/tag/2024.08.0-77) - which includes Quarto extension 1.114.0
* Open .qmd file, and try to preview

```
C:\Users\JonVanausdeln\source\qa-example-content [main ≡]> quarto preview c:/Users/JonVanausdeln/source/qa-example-content/workspaces/quarto_basic/quarto_basic.qmd --no-browser --no-watch-inputs --output-dir C:\Users\JONVAN~1\AppData\Local\Temp\tmp-32232-hHeUVdON8Tb2 --embed-resources
quarto: The term 'quarto' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

**Comment by jmcphers** [comment]
This should work now _unless_ pyenv is active. As noted in https://github.com/quarto-dev/quarto/issues/517#issuecomment-2318143819, pyenv is removing Quarto from the `$PATH` which leads to the issue @jonvanausdeln describes above.

QA, can we close this one (assuming the non-pyenv case works okay)? I'll open a new issue to track the pyenv work.

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2024.09.0-1
    OS Version          : OSX

### Test scenario(s) 

Non pyenv runs work fine!

Link(s) to TestRail test cases run or created:
N/A
